membership/founding-membership.md

```diff
@@ -1,57 +1,42 @@
+* set up a sustainable relationship with the market - helping different (types of) companies work together without a single point of failure (e.g. service designers with developers with system maintainers)
+* make sure you and other public members are in control of their systems so you can respond to new citizen needs, policy goals or technologies
+
+Further benefits of membership include being able to:
 
-* be one of the first to have codebases stewarded by the Foundation (reducing your future risk and maintenance costs)
-* get a seat on the board of directors
 * steer the Foundation’s pipeline towards projects you’d like to reuse
 * gain recognition as a leader of systemic digital transformation
-
-This is in addition to the regular benefits of Foundation membership. These include being able to:
-
-* invoke continuous integration auditing resources on projects in development accepted by the Foundation (increasing the security of your code)
 * demonstrate return on IT investments from higher levels of administration
-* collectively participate in a trusted Foundation procurement strategy
+* increasing the security and quality of your code through continuous auditing resources
+* access Foundation leaders and advisors for open source strategy discussions
 * learn from and share experience with other bold, innovative public organizations and leaders in the sector
 
-We also offer:
-
-* access to Foundation leaders and advisors for open source strategy discussions
-* entry to exclusive, invite-only public code leadership summits focused on technical, compliance, and business topics
-* prominent placement of your organization’s brand on the Foundation’s social networks and web properties
-* direct access to technical decision-makers of the Foundation platform
-* opportunities for leadership roles in Foundation initiatives
-* a blogpost announcing membership
+## Membership requirements
 
-## Founding membership requirements
-
-To become a founding member you should have both of these:
+To become a member you should be a publicly owned organization (such as a city, regional government, public library or an association of public bodies), and have some of these:
 
 * enthusiasm for public code at the most senior levels within your organisation
-* the ability to make in-kind contributions to help us develop our processes and governance, so that they work better for your type of organisation
-
-To become a founding member you should have some of these:
-
-* a willingness to help the Foundation test its onboarding and code auditing processes, so we can get it right
+* a willingness to help improve the Foundation onboarding and code auditing processes
 * the ability to make a significant financial commitment relative to your type and size of organization (especially if you don’t currently have a codebase available, but are invested in creating a future for public code)
```

Comment:
suggest swapping this with the point underneath, as someone reading chronologically doesn't know yet that they're supposed to have a codebase

Reply:
done!